src/components/Request/NewRequestForm.tsx

```diff
@@ -71,12 +77,17 @@ export type RHFRequest = {
 };
 
 const NewRequestForm = () => {
+  const params = useParams();
+  const request = useRequest(Number(params.requestId));
   const addRequest = useMutateRequest();
 
   const today = new Date(Date.now());
+  // Remove optional Author and Created properties from data object
+  const { Author, Created, ...data } = request.data ?? {};
 
   const myForm = useForm<RHFRequest>({
     defaultValues: {
+      ...data,
```

Comment:
Not sure about what this is providing?  Won't most of the data elements be overwritten by the "blank" values specified below?  The defualt values will be reset to the values of the "Draft" item when it is called and has both data and a requestId by the useEffect